imageprocessor/src/main/java/pl/edu/uj/kimage/processingFlow/FlowStepRepository.java

```diff
@@ -0,0 +1,21 @@
+package pl.edu.uj.kimage.processingFlow;
+
+import java.util.Collection;
+import java.util.SortedMap;
+import java.util.concurrent.ConcurrentSkipListMap;
+
+public class FlowStepRepository {
+    private final SortedMap<Integer, FlowStep> flowSteps = new ConcurrentSkipListMap<>();
+
+    public void saveFlowStep(FlowStep flowStep) {
+        flowSteps.put(flowStep.getStepId(), flowStep);
+    }
+
+    public FlowStep load(int stepId) {
+        return flowSteps.get(stepId);
+    }
+
+    public Collection<FlowStep> loadAllHigherThen(int stepId){
+        return flowSteps.tailMap(stepId).values();
+    }
```

Comment:
I think it is too early to write this kind of code. Perhaps in the future you change your mind and leave the unused code in the project